tests/vxlan/test_vnet_vxlan.py

```diff
@@ -0,0 +1,167 @@
+    ptfhost.copy(content=arp_responder_conf, dest="/etc/supervisor/conf.d/arp_responder.conf")
+
+    ptfhost.shell("supervisorctl reread")
+    ptfhost.shell("supervisorctl update")
+
+    logger.debug("VNet config is: " + str(vc.VNET_CONFIG))
+    vnet_json = {
+        "minigraph_port_indices": mg_facts["minigraph_port_indices"],
+        "minigraph_portchannel_interfaces": mg_facts["minigraph_portchannel_interfaces"],
+        "minigraph_portchannels": mg_facts["minigraph_portchannels"],
+        "minigraph_lo_interfaces": mg_facts["minigraph_lo_interfaces"],
+        "minigraph_vlans": mg_facts["minigraph_vlans"],
+        "minigraph_vlan_interfaces": mg_facts["minigraph_vlan_interfaces"],
+        "dut_mac": dut_facts["ansible_Ethernet0"]["macaddress"],
+        "vnet_interfaces": vc.VNET_CONFIG["vnet_intf_list"],
+        "vnet_routes": vc.VNET_CONFIG["vnet_route_list"],
+        "vnet_local_routes": vc.VNET_CONFIG["vnet_local_routes"],
+        "vnet_neighbors": vc.VNET_CONFIG["vnet_nbr_list"],
+        "vnet_peers": vc.VNET_CONFIG["vnet_peer_list"]
+    }
+    ptfhost.copy(content=json.dumps(vnet_json, indent=2), dest="/tmp/vnet.json")
+
+@pytest.fixture(scope="module")
+def setup(duthost, ptfhost, num_vnet, num_routes, num_endpoints, ipv6_vxlan_test, skip_cleanup):
+    """
+    @summary: Fixture to prepare the DUT and PTF hosts for testing
+    @param duthost: DUT host object
+    @param ptfhost: PTF host object
+    @param num_vnet: Number of VNETs
+    @param num_routes: Number of routes
+    @param num_endpoints: Number of endpoints
+    @param ipv6_vxlan_test: Use IPV6 for testing
+    @param skip_cleanup: Determines if cleanup is skipped or not
+    """
+    minigraph_facts = duthost.minigraph_facts(host=duthost.hostname)["ansible_facts"]
+    dut_facts = duthost.setup(gather_subset="!all,!any,network", filter="ansible_Ethernet*")["ansible_facts"]
+
+    vc.IPV6_VXLAN_TEST = ipv6_vxlan_test
+
+    vc.CLEANUP = not skip_cleanup
+
+    gen_vnet_config(minigraph_facts, num_vnet, num_routes, num_endpoints)
+
+    prepare_ptf(ptfhost, minigraph_facts, dut_facts)
+
+    generate_dut_config_files(duthost, minigraph_facts)
+
+    yield minigraph_facts 
+
+@pytest.fixture(params=["Disabled", "Enabled", "Cleanup"])
+def vxlan_status(setup, request, duthost):
+    """
+    @summary: Paramterized fixture that tests the Disabled, Enabled, and Cleanup configs for VxLAN
+    @param setup: Pytest fixture that provides access to minigraph facts
+    @param request: Contains the parameter (Disabled, Enabled, or Cleanup) for the current test iteration
+    @param duthost: DUT host object
+    @param returns: VxLAN status, and the test scenario
+    """
+    vxlan_enabled = False
+    if request.param == "Disabled":
+        vxlan_enabled = False
+    elif request.param == "Enabled":
+        mg_facts = setup
+
+        duthost.shell("sonic-clear fdb all")
+        
+        attach_to = mg_facts["minigraph_vlan_interfaces"][0]['attachto']
```

Comment:
I would give this a more descriptive name (e.g. is "attach_to" the VLAN name or something else?).